posthog/session_recordings/sql/session_replay_event_sql.py

```diff
@@ -72,6 +78,8 @@
     event_count SimpleAggregateFunction(sum, Int64),
     -- which source the snapshots came from Android, iOS, Mobile, Web. Web if absent
     snapshot_source AggregateFunction(argMin, LowCardinality(Nullable(String)), DateTime64(6, 'UTC')),
+    -- knowing something is mobile isn't enough, we need to know if e.g. RN or flutter
+    snapshot_library AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC')),
```

Comment:
from a clickhouse review perspective - at a minimum we want to be able to do usage reports for billing without joining on the events table to get the library for the different mobile platforms

but this will also let us do some nice query and display things